docs/presentation/Content/ContentInstancesOfSpecificClasses.md

```diff
@@ -1,39 +1,178 @@
-# ContentInstancesOfSpecificClasses Specification
+# Content Instances Of Specific Classes Specification
 
 > TypeScript type: [ContentInstancesOfSpecificClassesSpecification]($presentation-common).
 
-Returns content for instances of specific ECClasses.
+A specification that creates content for  instances of specific ECClasses.
```

Comment:
The problem that I see with **all** is that it's kind of a lie - as you said yourself, it's not *always* and in practice I think it's actually a rare case to use this specification without any additional filtering, in which case it doesn't load all instances. Maybe an additional sentence like "All instances of requested classes are loaded unless additional filtering is used" could be added.

Reply:
I prefer @roluk version since, unless specified in the filter, it does select all instances of the target classes.
@grigasp mentioned that it's usually used with some filter, but the filter is not mandatory so I don't think adding `"All instances of requested classes are loaded unless additional filtering is used"` is very accurate either.